core-bundle/contao/widgets/RootPageDependentSelect.php

```diff
@@ -33,7 +33,7 @@ public function generate(): string
 			$this->strLabel = $rootPage->title;
 
 			$fields[] = \sprintf(
-				'<select name="%s[]" id="ctrl_%s" class="tl_select%s"%s data-action="focus->contao--scroll-offset#store" data-controller="contao--choices">%s</select>%s',
+				'<div data-controller="contao--choices"><select name="%s[]" id="ctrl_%s" class="tl_select%s"%s data-action="focus->contao--scroll-offset#store">%s</select></div>%s',
```

Comment:
Is the wizard still displayed correctly now that there is a block element wrapped around the SELECT?

Reply:
Improved in 960ae9a64ec2ba903fc0df80096c70f7c659782b